### ISSUE TITLE
Typo in non-softmax normalizations

### ISSUE THREAD

**Issue by DNGros** [thomlake/pytorch-attention#1]
It looks like there is typo in https://github.com/thomlake/pytorch-attention/blob/31059b18fd222db9afb04e186a9e06dd203880af/attention/attention.py#L265-L266 . You probably want to use the compare `normalize` with the strings, not `w`. This is a fairly minor issue since attention pretty much always uses softmax normalization
